src/RouterSelector.web.js

```diff
@@ -161,6 +162,17 @@ const RouterSelector = () => {
     initAnalytics()
   }, [])
 
+  useEffect(() => {
+    //once user is logged in check if their browser is supported and show warning if not
+    if (isLoggedIn) {
+      setCheckedForBrowserSupport(true)
+      checkBrowser()
+    } else {
+      setIgnoreUnsupported(true)
+      setCheckedForBrowserSupport(true)
+    }
+  }, [isLoggedIn])
+
```

Comment:
`setCheckedForBrowserSupport` starts animation. which isn't needed if we will check the browser (because animation time aligned with router initialization promises (they have Promise.race with delay(animationDuration)

```suggestion
  useEffect(() => {
    // once user is logged in check if their browser is supported and show warning if not
    if (isLoggedIn) {
      checkBrowser()
      return
    }
    
    // start initialize router
    setIgnoreUnsupported(true)
    // show animation
    setCheckedForBrowserSupport(true)    
  }, [isLoggedIn])

```

Reply:
shouldnt     setIgnoreUnsupported(true) be called only when we want to continue? 
that is how  it was used previously.
setCheckedForBrowserSupport was called after we tested the browser to trigger the splash animation
setIgnoreUnsupported was called after user chose to continue to trigger the actuall app loading.

I guess they could be both set to true to do everything  in the background while user is show the dialog...

Reply:
hmm. then let's remove them both and also useBrowserSupport from here. let the app initializes as usual.
if user is logged he will come to the Dashboard. so let's useBrowserSupport from the dashboard

Reply:
i prefer to leave it here. dashboard ttakes long time to load